CONTRIBUTING.md

```diff
@@ -104,7 +104,12 @@ Basically, the `translate()` method should return a `String` for the compiler to
 ### 4. Tests
 Amber uses `cargo test` for tests. `stdlib` and `validity` tests usually work by executing amber code and checking its output.
 
-We have [`validity tests`](src/tests/validity.rs) to check the compiler, [`stdlib tests`](src/tests/stdlib.rs) and [`CLI tests`](src/tests/cli.rs).
+We have [`validity tests`](src/tests/validity.rs) to check if the compiler outputs a valid bash code, [`stdlib tests`](src/tests/stdlib.rs) and [`CLI tests`](src/tests/cli.rs).  
+
+The majority of `stdlib` tests are Written in pure Amber in the folder [`tests/stdlib`](src/tests/stdlib). For every test there is a `*.output.txt` file that contains the expected output.
+Tests will be executed without recompilation. Amber will load the scripts and verify the output in the designated file to determine if the test passes.
+
+Some tests, such as those for `download`, require Rust to load a web server. Therefore, there is a separate folder [`tests/stdlib/no_output`](src/tests/stdlib/no_output) that contains only the Amber script. The output is located in the [`stdlib tests`](src/tests/stdlib.rs) file.
```

Comment:
some tests require a web server -> so there is a separate folder for them

is it just me or there is no logic here?

Reply:
I was thinking that it was clear that some requires some other stuff.

Reply:
@b1ek you're right. It feels off. Let's change it to:
```suggestion
Some tests require additional setup, such as those for `download` that needs Rust to load a web server. These functions require special tests written in Rust that we can find in [`stdlib tests`](src/tests/stdlib.rs) file. The designated directory where to store the amber files is located in [`tests/stdlib/no_output`](src/tests/stdlib/no_output). These tests do not coexist with `.output.txt` files hence the name of this folder.
```